### ISSUE TITLE
SaaS subscription plan names

### ISSUE THREAD

**Issue by beninsydney** [faker-js/faker#883]
### Clear and concise description of the problem

A generator for SaaS subscription plan names, like 'beginner', 'pro', 'gold' 'silver', 'bronze' or 'platinum' type stuff.

### Suggested solution

    faker.commerce.plan()

**Comment by Shinigami92** [comment]
Current workaround: `faker.random.arrayElement(['beginner', 'pro', 'gold' 'silver', 'bronze' or 'platinum'])`

**Comment by wael-fadlallah** [comment]
I can add this method to the commerce module if it's agreed upon

**Comment by ST-DDT** [comment]
I'm not sure whether this is useful.
If you are a provider for a service you will usually use your own subscription names.

Could you please describe your use-case a bit more?

**Comment by ndianabasi** [comment]
> If you are a provider for a service you will usually use your own subscription names with `random.arrayElement`.

I agree with this. I don't see any additional benefits in creating a specialised method for subscription plans.

**Comment by beninsydney** [comment]
> @beninsydney Could you please describe your use-case a bit more?

Creating mockups of subscription software, plan selection screens etc where the plans are fictional or TBD, I use faker for realistic customer information but the existing support for products (faker.commerce.product()) is physical goods rather than online plans.
